amd_openvx_extensions/amd_rpp/include/internal_publishKernels.h

```diff
@@ -116,86 +124,93 @@ vx_status VignettebatchPD_Register(vx_context);
-#define VX_KERNEL_RPP_COLORTEMPERATUREBATCHPD_NAME      		"org.rpp.ColorTemperaturebatchPD"
-#define VX_KERNEL_RPP_VIGNETTEBATCHPD_NAME      				"org.rpp.VignettebatchPD"
-#define VX_KERNEL_RPP_CHANNELEXTRACTBATCHPD_NAME      			"org.rpp.ChannelExtractbatchPD"
-#define VX_KERNEL_RPP_CHANNELCOMBINEBATCHPD_NAME      			"org.rpp.ChannelCombinebatchPD"
-#define VX_KERNEL_RPP_LOOKUPTABLEBATCHPD_NAME      				"org.rpp.LookUpTablebatchPD"
-#define VX_KERNEL_RPP_BOXFILTERBATCHPD_NAME      				"org.rpp.BoxFilterbatchPD"
-#define VX_KERNEL_RPP_SOBELBATCHPD_NAME      					"org.rpp.SobelbatchPD"
-#define VX_KERNEL_RPP_MEDIANFILTERBATCHPD_NAME      			"org.rpp.MedianFilterbatchPD"
-#define VX_KERNEL_RPP_CUSTOMCONVOLUTIONBATCHPD_NAME      		"org.rpp.CustomConvolutionbatchPD"
-#define VX_KERNEL_RPP_NONMAXSUPRESSIONBATCHPD_NAME      		"org.rpp.NonMaxSupressionbatchPD"
-#define VX_KERNEL_RPP_GAUSSIANFILTERBATCHPD_NAME      			"org.rpp.GaussianFilterbatchPD"
-#define VX_KERNEL_RPP_NONLINEARFILTERBATCHPD_NAME      			"org.rpp.NonLinearFilterbatchPD"
-#define VX_KERNEL_RPP_LOCALBINARYPATTERNBATCHPD_NAME      		"org.rpp.LocalBinaryPatternbatchPD"
-#define VX_KERNEL_RPP_DATAOBJECTCOPYBATCHPD_NAME      			"org.rpp.DataObjectCopybatchPD"
-#define VX_KERNEL_RPP_GAUSSIANIMAGEPYRAMIDBATCHPD_NAME      	"org.rpp.GaussianImagePyramidbatchPD"
-#define VX_KERNEL_RPP_LAPLACIANIMAGEPYRAMID_NAME      			"org.rpp.LaplacianImagePyramid"
-#define VX_KERNEL_RPP_CANNYEDGEDETECTOR_NAME      				"org.rpp.CannyEdgeDetector"
-#define VX_KERNEL_RPP_HARRISCORNERDETECTOR_NAME      			"org.rpp.HarrisCornerDetector"
-#define VX_KERNEL_RPP_FASTCORNERDETECTOR_NAME      				"org.rpp.FastCornerDetector"
-#define VX_KERNEL_RPP_REMAP_NAME      							"org.rpp.remap"
-#define VX_KERNEL_RPP_TENSORADD_NAME      						"org.rpp.TensorAdd"
-#define VX_KERNEL_RPP_TENSORSUBTRACT_NAME      					"org.rpp.TensorSubtract"
-#define VX_KERNEL_RPP_TENSORMULTIPLY_NAME      					"org.rpp.TensorMultiply"
-#define VX_KERNEL_RPP_TENSORMATRIXMULTIPLY_NAME      			"org.rpp.TensorMatrixMultiply"
-#define VX_KERNEL_RPP_TENSORLOOKUP_NAME      					"org.rpp.TensorLookup"
-#define VX_KERNEL_RPP_COLORTWISTBATCHPD_NAME        			"org.rpp.ColorTwistbatchPD"
-#define VX_KERNEL_RPP_CROPMIRRORNORMALIZEBATCHPD_NAME        	"org.rpp.CropMirrorNormalizebatchPD"
-#define VX_KERNEL_RPP_CROPPD_NAME   							"org.rpp.CropPD"
-#define VX_KERNEL_RPP_RESIZECROPMIRRORPD_NAME      				"org.rpp.ResizeCropMirrorPD"
-#define VX_KERNEL_RPP_SEQUENCEREARRANGE_NAME                    "org.rpp.SequenceRearrange"
+// Tensor kernel names
+#define VX_KERNEL_RPP_NOPTENSOR_NAME                                         "org.rpp.NopTensor"
+#define VX_KERNEL_RPP_COPYTENSOR_NAME                                        "org.rpp.CopyTensor"
+#define VX_KERNEL_RPP_BRIGHTNESS_NAME                                        "org.rpp.Brightness"
+#define VX_KERNEL_RPP_GAMMACORRECTION_NAME                                   "org.rpp.GammaCorrection"
```

Comment:
@fiona-gladwin can we make the names uniform and add tensor to brightness and gamma correction as well?